tests/src/application/controllers/forget-password.spec.ts

```diff
@@ -67,13 +77,7 @@ describe('ForgetPassword Controller', () => {
   it('Should return 400 if an invalid email is provided', async () => {
     const { sut, emailValidatorStub } = makeSut()
     jest.spyOn(emailValidatorStub, 'isValid').mockReturnValueOnce(false)
-    const httpRequest = {
-      body: {
-        email: 'invalid_email@mail.com'
-      }
-    }
-
-    const httpResponse = await sut.handle(httpRequest)
+    const httpResponse = await sut.handle(makeFakeRequest())
     expect(httpResponse.statusCode).toBe(400)
```

Comment:
and if here we test the return completely, checking not only the status but the http helper that includes the `statusCode` and the type of error. What do you think? There are some examples in the code with these implementations.